jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/ContainerActions.kt

```diff
@@ -190,3 +185,11 @@ class ExecuteCommandInShellAction(
             pluginIsInstalledAndEnabled("org.jetbrains.plugins.terminal") && AwsToolkit.isEcsExecEnabled()
     }
 }
+
+object CurrentConnectionSettings {
```

Comment:
This feels like an odd home for this method

We should probably make ConnectionSettings.toEnvironmentVariables() be an extension method as well

Reply:
Added an Extension function to ConnectionSettings